streams/conversion.ts

```diff
@@ -196,6 +196,78 @@ export function readableStreamFromIterable<T>(
   });
 }
 
+/**
+ * Convert the generator function into a TransformStream.
+ *
+ * ```ts
+ * import { readableStreamFromIterable, transformStreamFromGeneratorFunction } from "https://deno.land/std@$STD_VERSION/streams/mod.ts";
```

Comment:
This should probably use relative import like other APIs above

Reply:
Thank you for your review. Has been updated.